lib/seattleflu/db/cli/command/etl/clinical.py

```diff
@@ -224,28 +225,41 @@ def encounter_details(document: dict) -> dict:
         }
 
 
-def race(race_name: str) -> list:
+def race(races: str or list) -> list:
```

Comment:
This isn't a valid type annotation, although I don't blame you at all for not knowing since our type-checking for this repo is in disrepair. What you want is called a "union" type, which syntactically is `Union[str, list]`. `Union` comes from the core `typing` module.

(I really need to get type-checking sorted out so that we can actually get some automatic benefit from our type annotations, but it always seems so low priority.)

Reply:
Thanks for the tip! I would love to learn more about type-checking in python.

Reply:
Happy to chat more about this in person at any time. :-)